### PR TITLE
net/wireguard: introduce warning before going stable

### PR DESCRIPTION
**Pull request by mimugmail**
0.10 will be for 19.1.7 as a devel and CFT .. when there are no show stoppers we can include it for stable in 19.1.8

### CODE REVIEW

**Comment by L1ghtn1ng** [comment]
@mimugmail You have clamav things in this PR, can you remove please? Then should be ready for @fichtner after that

File: net/wireguard/src/opnsense/mvc/app/views/OPNsense/Wireguard/general.volt [hunk 1]
```diff
@@ -36,6 +36,10 @@ POSSIBILITY OF SUCH DAMAGE.
     <li><a data-toggle="tab" href="#showhandshake">{{ lang._('Handshakes') }}</a></li>
 </ul>
 
+<div class="alert alert-warning" role="alert" style="min-height:65px;">
+  <div style="margin-top: 8px;">{{ lang._('This software is still in experimental state, use with caution.') }}</div>
```
Comment (fichtner) [inline net/wireguard/src/opnsense/mvc/app/views/OPNsense/Wireguard/general.volt]:
```suggestion
  <div style="margin-top: 8px;">{{ lang._('The WireGuard VPN software is still in experimental state, use with caution.') }}</div>
```

**Comment by mimugmail** [comment]
This is only a merge of master, no Changes to clamav code :)

**Comment by fichtner** [comment]
Merged, thanks!

**Pull request merged**